ROADMAP.md

```diff
@@ -33,6 +37,10 @@ Add support for [HMPP](https://kotlinlang.org/docs/multiplatform-share-on-platfo
 
 As the new [memory model](https://blog.jetbrains.com/kotlin/2021/08/try-the-new-kotlin-native-memory-manager-development-preview/) is becoming the default, adapt Apollo Kotlin to use it.
```

Comment:
We can remove the HMPP and new memory model sections, as this is already done 🎉

Reply:
Thanks! I'll make that change.